content/en/docs/setup/production-environment/tools/kubeadm/create-cluster-kubeadm.md

```diff
@@ -312,12 +312,11 @@ kubectl apply -f https://docs.projectcalico.org/v3.14/manifests/calico.yaml
 {{% /tab %}}
 
 {{% tab name="Cilium" %}}
-For Cilium to work correctly, you must pass `--pod-network-cidr=10.217.0.0/16` to `kubeadm init`.
```

Comment:
hi, is this no longer needed?
can you show me a link where this is stated?

Reply:
Hi, @neolit123 correct this is no longer needed it as Cilium provides its own IPAM module that is capable to manage pod's network automatically.

Reply:
awesome. do you have a link for the implementation / docs for this on the Cillium side?

Reply:
@neolit123 yes, here: https://docs.cilium.io/en/latest/gettingstarted/ipam-cluster-pool/

Reply:
very nice.

/lgtm